components/price/price.js

```diff
@@ -98,6 +120,12 @@ Price.defaultProps = {
 
 Price.propTypes = {
   /**
+   * Additional text that will be displayed under price and the line.
+   * E.g.: 'per day'.
+   */
+  additionalText: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
```

Comment:
@reaktivo yes, you're right... didn't know that... But next time, add some factual documentation links to it: https://facebook.github.io/react/docs/typechecking-with-proptypes.html#proptypes

Doesn't hurt much 👍